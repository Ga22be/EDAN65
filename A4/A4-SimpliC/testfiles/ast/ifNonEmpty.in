int a(int house, int boat) {
    if (house < boat) {
        int a;
    }
}