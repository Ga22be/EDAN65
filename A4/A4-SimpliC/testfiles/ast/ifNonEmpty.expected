Program
  List
    Function
      IdDecl ID="a"
      List
        Parameter
          IdDecl ID="house"
        Parameter
          IdDecl ID="boat"
      List
        IfStatement
          LesserThan
            IdUse ID="house"
            IdUse ID="boat"
          List
            Declaration
              IdDecl ID="a"
          Opt
